src/pyhf/__init__.py

```diff
@@ -17,7 +15,7 @@ def get_backend():
     Example:
         >>> import pyhf
         >>> pyhf.get_backend()
-        (<pyhf.tensor.numpy_backend.numpy_backend object at 0x...>, <pyhf.optimize.opt_scipy.scipy_optimizer object at 0x...>)
+        (<pyhf.tensor.numpy_backend.numpy_backend object at 0x...>, <pyhf.optimize.scipy_optimizer object at 0x...>)
```

Comment:
Probably helps to make this more explicit now (I can't edit the `pyhf.get_backend()` above so doing so here, but would need to get fixed up so I can also do this one)
```suggestion
        backend, optimizer = pyhf.get_backend()
        >>> backend
        <pyhf.tensor.numpy_backend.numpy_backend object at 0x...>
        >>> optimizer
        <pyhf.optimize.scipy_optimizer object at 0x...>
```